meta-balena-common/recipes-support/ca-certificates/ca-certificates/extract-balena-ca

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+CONFIG_JSON="/mnt/boot/config.json"
+CA_BUNDLE="/etc/ssl/certs/ca-certificates.crt"
+JSON_CA_KEY=".balenaRootCA"
+BALENA_CA_FILE="/usr/share/ca-certificates/balena/balenaRootCA.crt"
+
+log () {
```

Comment:
Consider having homogeneous logging by sourcing  `os-helpers-logging`. It's only used in the initramfs at the moment, but it may be good to start using it in new shell scripts so that the output looks the same for all.

Reply:
+1 I was not aware of that. `os-helpers` are not installed in the OS right now, are you OK with pulling it in with this PR?

Reply:
I think it's a good moment. It won't add a lot of space and it will help to make the output of script homogeneous.